kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -39,25 +26,24 @@ def content_renderer_frontend_async():
 
 
 @register.simple_tag()
-def kolibri_main_navigation():
+def initialized_data():
```

Comment:
Just a heads-up, I'd noticed an issue with the urls where they change with every refresh (prepending a directory). My guess is maybe some code is running with every page load which was only intended to run on app initialization or something?

It ends up generating data that looks like:

```javascript
{"user_nav_items": [{"url": "management/management/management/management/management/management/management/management/management/management/management/management/learners", "text": "Learner Management"}], "nav_items": [{"url": "learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/", "text": "Learn!"}, {"url": "learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/learn/foo", "text": "Learn foo!"}, {"url": "management/management/management/management/management/management/management/management/management/management/management/management/foo/bar", "text": "Management Foo!"}]}
```